src/js/components/DataTable/stories/typescript/Styled.tsx

```diff
@@ -28,5 +28,5 @@ const StyledDataTable = () => (
 );
 
 if (!isChromatic()) {
-  storiesOf('DataTable', module).add('Styled', () => <StyledDataTable />);
+  storiesOf('DataTable', module).add('Styled TS', () => <StyledDataTable />);
```

Comment:
```suggestion
  storiesOf('TypeScript/DataTable', module).add('Style', () => <StyledDataTable />);
```
I think maybe the fact that the directory is missing caused the warning here.